hmp/transform.py

```diff
@@ -49,7 +49,7 @@ def __str__(self) -> str:
         return self.value
 
     def __bool__(self) -> bool:
-        return self != cls.NO_ANALYSIS
+        return self != cls.DONT_APPLY
```

Comment:
This was a mistake, but somehow `method=None` or `method='no_analysis'` doesn't work